### DISCUSSION TITLE
argo-workflows/rest-examples/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#11226] [General]
# argo-workflows/rest-examples/

https://argoproj.github.io/argo-workflows/rest-examples/

**Comment by joebowbeer** [comment]
What is the difference between the namespace in the url and the namespace in the submitted data?

**Comment by vineetkala11** [comment]
All the above given apis are to Create, Get and Delete workflow. Is there an api for triggering existing workflow?

**Reply by joebowbeer** [reply]
Yes, and it is not well documented.

POST to `submit` endpoint and reference a WorkflowTemplate. A reference to a Workflow might work, too.

Some discussion here: https://github.com/argoproj/argo-workflows/discussions/9025

**Comment by haochih** [comment]
I have no access to CRD files of Argo Workflows installation.
How to find out whether an Argo Workflows Controler/Server is in cluster-scope or namespaced from Web UI?

Thanks

**Comment by LiteshSinha** [comment]
Is it possible to modify the sensor to share an available pod when multiple events occur within a certain time frame range, instead of creating a new pod for each event when triggering 10 events in 1 second?

**Comment by vidhi-p-agrawal** [comment]
Can I have an  API example of submitting cronworkflow

**Comment by sprite-91** [comment]
If I set `workflows.argoproj.io/creator: abc` in `labels`, why is the creator in the submitted workflow set to `system-serviceaccount-argo-argo-server`? 
This problem will occur in `api/v1/workflows/{namespace}`, but not in `api/v1/workflows/{namespace}/submit`.

```sh
curl --request POST \
  --url https://localhost:2746/api/v1/workflows/argo \
  --header 'content-type: application/json' \
  --data '{
  "namespace": "argo",
  "serverDryRun": false,
  "workflow": {
      "metadata": {
        "generateName": "hello-world-",
        "namespace": "argo",
        "labels": {
          "workflows.argoproj.io/completed": "false",
          "workflows.argoproj.io/creator": "abc"
         }
      },
     "spec": {
       "templates": [
        {
         "name": "whalesay",
         "arguments": {},
         "inputs": {},
         "outputs": {},
         "metadata": {},
         "container": {
          "name": "",
          "image": "docker/whalesay:latest",
          "command": [
            "cowsay"
          ],
          "args": [
            "hello world"
          ],
          "resources": {}
        }
      }
    ],
    "entrypoint": "whalesay",
    "arguments": {}
  }
}
}'
```

**Comment by vkunniyur** [comment]
Are there examples of how to fetch the status code of the steps that are part of argo workflows?
